nginx/etc/nginx/nginx.conf

```diff
@@ -21,7 +21,12 @@ http {
 
     access_log /var/log/nginx/access.log timed_combined;
 
-    keepalive_timeout 65;
+    keepalive_disable msie6;
+    keepalive_timeout 65s;
```

Comment:
This needs to be greater than the ALB idle timeout and the upstream request timeout.